index.js

```diff
@@ -443,8 +443,10 @@ const getPodcastFromURL = exports.getPodcastFromURL = async function (url, param
     const feedResponse = await fetchFeed(url)
     const channel = feedResponse.rss.channel[0]
 
-    if (channel["itunes:new-feed-url"]) {
-      return await getPodcastFromURL(channel["itunes:new-feed-url"][0], params)
+    const newURL = channel["itunes:new-feed-url"][0]
```

Comment:
If there is no property `"itunes:new-feed-url"`, accessing the 0th element will fail.